### PR TITLE
Fix and improve "Show PID/TID in HEX" feature

### PR DESCRIPTION
**Pull request by not6**
There's still a lot of improvements that could be made, where the PID/TID is always hexadecimal, but this was the worst one IMO.

### CODE REVIEW

File: src/dbg/debugger.cpp [hunk 1]
```diff
@@ -482,7 +483,10 @@ static void DebugUpdateTitle(duint disasm_addr, bool analyzeThreadSwitch)
         {
             char threadName2[MAX_THREAD_NAME_SIZE] = "";
             if(!ThreadGetName(PrevThreadId, threadName2) || threadName2[0] == 0)
-                sprintf_s(threadName2, "%X", PrevThreadId);
```
Comment (mrexodia) [inline src/dbg/debugger.cpp:485]: There is a function `formatpidtid` that does this

File: src/dbg/debugger.cpp [hunk 2]
```diff
@@ -492,7 +496,7 @@ static void DebugUpdateTitle(duint disasm_addr, bool analyzeThreadSwitch)
     if(ThreadGetName(currentThreadId, threadName) && *threadName)
         strcat_s(threadName, " ");
     char PIDnumber[64], TIDnumber[64];
-    if(settingboolget("Gui", "PidInHex"))
```
Comment (mrexodia) [inline src/dbg/debugger.cpp:495]: Same, it can be switched to the function `formatpidtid`

**Review by mrexodia — CHANGES_REQUESTED** [review]

**Comment by not6** [comment]
@mrexodia Should be good now

**Review by mrexodia — APPROVED** [review]

**Comment by mrexodia** [comment]
Awesome, thanks a lot!

**Pull request merged**